src/pages/info.tsx

```diff
@@ -45,53 +45,69 @@ const Title = styled.div`
   }
 `
 
-const HeaderLink = styled(Link)`
-  background-color: ${props => props.theme.colors.backgroundLight};
-  color: ${props => props.theme.colors.text};
-  font-size: ${props => props => props.theme.sizes.buttonText};
-  padding: 12px;
-  text-decoration: none;
-  border-radius: 12px;
-  display: flex;
-  align-items: center;
-  font-weight: 500;
-  box-shadow: 0 5px 20px rgba(0, 0, 0, 0.1);
+const Audience = styled.div`
+  flex: 1 1 600px;
+  font-size: ${props => props.theme.sizes.title / 2};
+  margin: 6px;
+
+  @media (max-width: ${mobileBreakpoint}px) {
+    font-size: ${props => props => props.theme.sizes.buttonText};
+  }
 `
 
-const ResultsPageContainer = styled.div`
+const InfoPageContainer = styled.div`
   overflow: auto;
 `
 
-export const ResultsPage: React.FC = () => {
+export const InfoPage: React.FC = () => {
   const query = useQuery()
   const { t } = useTranslation()
 
-  // parse /results?id=a,b,c -> [a, b, c]
+  // parse /info?id=a,b,c -> [a, b, c]
   const queryClasses = query.get('id')
   const classes = queryClasses ? queryClasses.split(',') : []
 
+  // build a comma separated, readable list of classes
+  var classString: string = ''
```

Comment:
```suggestion
  let classString: string = ''
```

var should _not_ be used when writing ES6